job_definitions/release_pipeline.yml

```diff
@@ -137,7 +137,7 @@
           git url: "git@github.com:alphagov/digitalmarketplace-{{ application }}.git",
               branch: 'master', credentialsId: 'github_com_and_enterprise', poll: true
           releaseNumber = sh(
```

Comment:
Is it still a `releaseNumber`?

Reply:
Good point! I have renamed it in bd0bab0